<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconocimiento Facial</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo-UNLa.jpg</normaloff>logo-UNLa.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 italic 12pt &quot;Times New Roman&quot;;
background-image: url(&quot;frameunla.png&quot;);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="main_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>331</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string> Sistema de Reconocimiento Facial</string>
       </property>
      </widget>
      <widget class="QFrame" name="ImgWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>511</width>
         <height>261</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingrese Nombre:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameText">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>320</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>320</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="startButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>320</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Iniciar</string>
       </property>
      </widget>
      <widget class="QLabel" name="mensajeLabel">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <weight>9</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHunla">
    <property name="title">
     <string>Menu Principal</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionInciar_Busqueda"/>
   </widget>
   <addaction name="menuHunla"/>
  </widget>
  <action name="actionInciar_Busqueda">
   <property name="text">
    <string>Inciar Busqueda</string>
   </property>
  </action>
  <action name="actionAgregar_Persona">
   <property name="text">
    <string>Agregar Imagen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
